Assets/Scripts/Models/Time/TimeManager.cs

```diff
@@ -198,6 +212,15 @@ public void Destroy()
     }
 
     /// <summary>
+    /// Runs the next frame before any updates are ran.
```

Comment:
.... I *think* you *may* be right... but they both sound just as *proper* when I say them, so I'm not entirely sure... 

And I'd also like to say I appreciate that even though I'm probably not going to change in this case (as I *think* both may be valid, and in either case am not certain which is correct), I appreciate that you've paid attention to the comments, since we've had issues with them being either inaccurate or just having terrible grammar/spelling.